### DISCUSSION TITLE
Synctex in web browser

### DISCUSSION THREAD

**Discussion by zingdle** [James-Yu/LaTeX-Workshop#3152] [Q&A]
**Is your feature request related to a problem? Please describe.**
I'm previewing PDF in web browser but the Synctex is broken in this mode. Is there's any plan to support this?

**Comment by tamuratak** [comment]
It is supported. https://github.com/James-Yu/LaTeX-Workshop/wiki/View#synctex

Fill out all the information required in the issue template. Please paste the whole log messages here, not parts of ones. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)
